packages/shader-lab/src/codeGen/VisitorContext.ts

```
@@ -70,52 +86,58 @@ export class VisitorContext {
-        ident.location
-      );
-    }
-    this._referencedVaryingList[lexeme] = props;
+  referenceVarying(ident: BaseToken): Error | void {
+    return this._referenceProp(
+      "varying",
+      ident.lexeme,
+      this.varyingList,
+      this._referencedVaryingList,
+      ident.location
+    );
```

ESLint

[error] Replace `⏎······"varying",⏎······ident.lexeme,⏎······this.varyingList,⏎······this._referencedVaryingList,⏎······ident.location⏎····` with `"varying",·ident.lexeme,·this.varyingList,·this._referencedVaryingList,·ident.location`

(prettier/prettier)